src/main.rs

```diff
@@ -504,19 +536,72 @@ fn print_results_stdout(times: &Vec<CommitHours>) -> Result<()> {
     Ok(())
 }
 
+fn print_breakdown_results_stdout(times: &Vec<CommitHours>) -> Result<()> {
```

Comment:
Is it easier to have a separate function to have the breakdown results? I would prefer to have only one function with a flag or similar. I'm not too well versed in the table formatter, but I was thinking maybe you could have a conditional that adds the extra conditional in breakdown mode. If it makes everything harder to read, I'm fine with this though.

Reply:
Good point, I'll think of a way for having a single function.